src/wallet/websocket.ts

```diff
@@ -0,0 +1,147 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import BaseWebSocket, {
+  WsOptions,
+  DEFAULT_WS_OPTIONS,
+} from '../websocket/base';
+
+export interface WalletServiceWebSocketOptions extends WsOptions {
+  walletId: string;
+};
+
+const JOIN_TIMEOUT = 5000;
```

Comment:
Should we include this in the `WalletServiceWebSocketOptions`?

Other timeouts are configurable through `WsOptions`

Reply:
I like the idea, done in a0e5091ee509a7a0e0e3672b7887973e4f845b33